CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## v1.6.0-rc1
 ### Note
 - Helm charts are changed to enforce PodDisruptionBudget for multi-replica clusters and PDB is removed from install manifests.
-
+- `anyPattern` of type `apiextensions.JSON` breaks for kyverno cluster policy CRD in k8s v1.23 but works for previous k8s versions due to a regression issue in v1.23 being tracked by this [PR](https://github.com/kubernetes/kubernetes/pull/107688) by kubernetes community. For context, please refer to this [issue #3004](https://github.com/kyverno/kyverno/issues/3004)
```

Comment:
`anyPattern` for Kyverno validate policies breaks in Kubernetes v1.23.0 and v1.23.1, and the fix is being tracked by this [PR](https://github.com/kubernetes/kubernetes/pull/107688) and will be available in v1.23.3.

Please specify `any`/`all` for policies that use `preconditions` and `deny.conditions`.
It would be good to add/link to an example of how to define these fields with any/all conditions.

Reply:
Thanks @realshuting , fixed the first part, in the latter part, would love if you can point out to some example policies to accordingly update, thanks a lot!

Reply:
This link should be good enough for telling the user how to use `any`/`all` conditions: https://kyverno.io/docs/writing-policies/preconditions/#any-and-all-statements

Reply:
Thanks a lot @zeborg, I will add the link!